### DISCUSSION TITLE
dnscrypt-proxy to cloudflare-gateway

### DISCUSSION THREAD

**Discussion by ng0177** [DNSCrypt/dnscrypt-proxy#1945] [Q&A]
dnscrypt-proxy does not accept

`server_names = ['nnnnnnnn.cloudflare-gateway.com/dns-query']`

in /etc/dnscrypt-proxy/dnscrypt-proxy.toml

Any plans for future integration?

**Comment by jedisct1** [comment]
The `server_names` parameter is a set of server... names :) Not a URL.

You can read the documentation on dnscrypt-proxy configuration [here](https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Configuration).

**Reply by ng0177** [reply]
very good documentation; however, stamps to `cloudflare-gateway` are not accepted ("invalid"). Any ideas?
